.changeset/sweet-sheep-film.md

```diff
@@ -0,0 +1,5 @@
+---
+"@rocket.chat/fuselage": patch
+---
+
+Updates the Field and InputBox component styles by changing their CSS tokens to follow new figma specifications
```

Comment:
```suggestion
feat(fuselage): update `InputBox` and `Field` typography
```

The pattern in this repo is to have the same PR title as the changeset description